src/development/environment-variables.md

```diff
@@ -3,40 +3,61 @@
 Platform.sh exposes environment variables which you can interact with. There are
 two sources for these. There are the ones that are set by Platform.sh itself
 and that give you all the context you need about the environment (how to
-connect to your database for example). And there are the ones you can set
-yourself through the web interface or the CLI.
+connect to your database, for example), and there are the ones you can set
+yourself.
 
-Environment variables are a good place to store values that apply only on Platform.sh, not on your local development environment. That includes API credentials for 3rd party services, mode settings if your application has a separate "Dev" and "Prod" toggle, etc.
+Environment variables are a good place to store values that apply only on Platform.sh and not on your local development environment. This includes API credentials for 3rd party services, mode settings if your application has a separate "Dev" and "Prod" toggle, etc.
 
-When you're logged in via SSH to an environment (with the cli: `platform ssh`),
-you can list the environment variables by running:
+## Adding environment variables
+
+New custom environment variables can be set [through the web interface](administration/web/configure-environment.html#settings), or using the CLI. For example, this command adds the variable `foo` to the `$PLATFORM_VARIABLES` object...
+
+```bash
+$ platform variable:set foo bar
+```
+## Inspecting and listing environment variables
+
+Custom environment variables which you have defined can be viewed [via the Web Interface](administration/web/configure-environment.html#settings) or using the CLI...
+
+```bash
+$ platform variables
+
++---------+-------+-----------+------+
+| ID      | Value | Inherited | JSON |
++---------+-------+-----------+------+
+| env:FOO | bar   | No        | No   |
++---------+-------+-----------+------+
```

Comment:
This says the name is `env:FOO`, but the previous section just named it `foo`.  That's highly confusing. Is it what actually happens?  It seems not from the text below.